exec/cnex/cnex.c

```diff
@@ -1310,9 +1364,54 @@ void exec_JUMPTBL(TExecutor *self)
     }
 }
 
-void exec_CALLX(void)
+void exec_CALLX(TExecutor *self)
 {
-    fatal_error("exec_CALLX not implemented");
+    self->ip++;
+    uint32_t mod = exec_getOperand(self);
+    uint32_t name = exec_getOperand(self);
+    uint32_t out_param_count = exec_getOperand(self);
+
+    char modname[256];
+    snprintf(modname, 256, "%s/%sneon_%s", path_getPathOnly(self->module->source_path), LIBRARY_NAME_PREFIX, self->module->bytecode->strings[mod]->data);
```

Comment:
All this seems like a lot of overhead to format up this shared library name on *every* CALLX opcode. Didn't we talk about having an array indexed by the `mod` index?

Reply:
We did, but this was sort of a "make it work quick" fix.  I can go back and move these to the TModule struct, if necessary.